### PR TITLE
Count votes in Propose Constitution test before ratification

### PR DESCRIPTION
**Pull request by carbolymer**
# Description
This PR consists of two commits:
* `Count votes in Propose Constitution test before ratification`
  This fixes failure of votes counting when testing for constitution ratification: https://github.com/IntersectMBO/cardano-node/actions/runs/8977554294/job/24656515973#step:13:959
* `Refactor: use singular nouns for module names` 
  Renames modules to use singular nouns

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated.  These may include:
  - golden tests
  - property tests
  - roundtrip tests
  - integration tests
  See [Runnings tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
- [ ] The version bounds in `.cabal` files are updated
- [ ] CI passes. See note on CI.  The following CI checks are required:
  - [ ] Code is linted with `hlint`.  See `.github/workflows/check-hlint.yml` to get the `hlint` version
  - [ ] Code is formatted with `stylish-haskell`.  See `.github/workflows/stylish-haskell.yml` to get the `stylish-haskell` version
  - [ ] Code builds on Linux, MacOS and Windows for `ghc-8.10.7` and `ghc-9.2.7`
- [ ] Self-reviewed the diff

# Note on CI
If your PR is from a fork, the necessary CI jobs won't trigger automatically for security reasons.
You will need to get someone with write privileges.  Please contact IOG node developers to do this
for you.

### CODE REVIEW

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/ProposeNewConstitution.hs [hunk 1]
```diff
@@ -198,20 +199,8 @@ hprop_ledger_events_propose_new_constitution = H.integrationWorkspace "propose-n
...
-    <- evalIO . runExceptT $ foldEpochState
-                      (File configurationFile)
-                      (File socketPath)
-                      FullValidation
-                      (EpochNo 10)
-                      ()
-                      (\epochState _ _ -> foldBlocksCheckConstitutionWasRatified constitutionHash constitutionScriptHash epochState)
-
-  void $ evalEither eConstitutionAdopted
+  _ <- waitForEpochs epochStateView (EpochInterval 1)
```
Comment (carbolymer) [inline cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/ProposeNewConstitution.hs:202]: I'm not sure if we should use `foldEpochState` to wait here for votes or simply wait for a single epoch. Single epoch wait seems easier and should be good enough imo.

Comment (Jimbo4350) [reply cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/ProposeNewConstitution.hs:202]: Can you add a comment explaining why we can be sure the proposal won't be removed from the ledger state after waiting an epoch?

**Review by Jimbo4350 — APPROVED** [review]
LGTM! One comment.

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/ProposeNewConstitution.hs (hunk 1 above)
Comment (carbolymer) [reply cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/ProposeNewConstitution.hs:202]: aded

**Pull request merged**